src/applications/model/four-elements-streaming-server.cc

```diff
@@ -45,14 +45,14 @@ FourElementsStreamingServer::GetTypeId (void)
 FourElementsStreamingServer::FourElementsStreamingServer ()
 {
   NS_LOG_FUNCTION (this);
-  InitializeStreams ();
+  SetReferenceBitRate (2.544);
 }
 
 FourElementsStreamingServer::FourElementsStreamingServer (Address ip, uint16_t port)
   : GamingStreamingServer (ip, port)
 {
   NS_LOG_FUNCTION (this << ip << port);
-  InitializeStreams ();
+  SetReferenceBitRate (2.544);
```

Comment:
This would make the codes very nice but it's not possible, look this:
https://stackoverflow.com/questions/2290733/initialize-parents-protected-members-with-initialization-list-c
I need to change the Constructor of the parent class to do that

Reply:
ok i didn't know, leave it as it is, but avoid using a trivial setter once you make the variable protected

Reply:
You mean I should ititialize m_referenceBitRate directly? (`m_referenceBitRate=2.544`) instead of calling `SetReferenceBitRate (2.544);` ?

Reply:
yes, there is no need for that function to exist IMO